ONSdigital/eq-author-app
# Fix ability to create knex migrations as per README

### What is the context of this PR?
- Fixes https://github.com/ONSdigital/eq-author-app/issues/117
- Removed as part of https://github.com/ONSdigital/eq-author-app/pull/74
- Moves `knexfile.js` to root so it's easier to resolve migrations directory
- Moves `settings.js` to root as `config` directory becomes redundant when it only contains `settings.js`
- Simplifies`yarn knex` command to remove unnecessary parameters
- Updates knex `README.md` commands to remove warning: 

> From Yarn 1.0 onwards, scripts don't require "--" for options to be forwarded. In a future version, any explicit "--" will be forwarded as-is to the scripts.

### How to review 
- Ensure you can run database migrations as per  [README](https://github.com/ONSdigital/eq-author-app/blob/master/eq-author-api/README.md) migration instructions
- Ensure you can successfully run `docker-compose up` and migrations are completed
